modopt/opt/gradient.py

```diff
@@ -33,6 +33,8 @@ class GradParent(object):
         Method for calculating the cost (default is ``None``)
     data_type : type, optional
         Expected data type of the input data (default is ``None``)
+    verbose : bool, optional, optional
+        Choose verbose mode
```

Comment:
You put `optional` twice, but perhaps to be more consistent with other parts of the code you can simply copy and paste this.

````python
'''
verbose : bool, optional
        Option for verbose output (default is ``True``)
'''
````

Reply:
Oh yep, sorry for that, I'll modify it